.rubocop.yml

```diff
@@ -42,3 +42,7 @@ SingleSpaceBeforeFirstArg:
 
 DotPosition:
   EnforcedStyle: trailing
+
+# Allow class and message or instance raises
+Style/RaiseArgs:
```

Comment:
I'm possibly misunderstanding, but wouldn't `raise IncompatibleSerializationError.new, transition_class.name` be fine?